packages/tools/devtools/devtools-browser-extension/webpack.config.js

```diff
@@ -80,5 +81,11 @@ module.exports = {
 				{ from: "**/*.html", to: ".", context: "src" },
 			],
 		}),
+		new Dotenv({
+			path: "./dev-example.env",
+			safe: true,
+			systemvars: true,
+			defaults: false,
```

Comment:
I think for the prod bundling config we don't want to override `safe`; there we only want the env variables set in the environment and anything in an `.env` file (which shouldn't exist in CI) to be used. For the same reason I would remove `path` here, and we can remove `defaults:  false` since that is the default value.

Let's also add a comment to clarify that `systemvars` is what causes the actual environment variables (not just from an `.env` file to be used.